Guidelines.md

```diff
@@ -0,0 +1,61 @@
+# Hydra Plugin Development Guidelines
+
+This document contains guidelines and expectations for developing Hydra plugins. It is one of the stated deliverables of the [Hydra Plugins Working Group](https://wiki.duraspace.org/display/hydra/
+Hydra+Plugins+Working+Group).
+
+These guidelines are supplementary, and subordinate, to existing guidelines that are documented elsewhere, including:
+  * [**General contributing guidelines**](https://github.com/projecthydra/hydra/blob/master/CONTRIBUTING.md)
+  * [**Project promotion process**](http://projecthydra-labs.github.io/promotion.html)
+
+
+## Architecture
+
+1. Plugins **must** be Ruby gems.
+
+1. Plugins **should** be Railties or Rails engines unless there is good reason to do otherwise.
```

Comment:
"Unless there is good reason to do otherwise" weakens the requirement a bit. What constitutes a good reason? What might some good reasons be? Is there an opportunity to be more explicit here about some of the considerations that go into choosing a Railtie vs. an engine vs. a plain ol' Ruby gem?